internal/jest/utils.js

```diff
@@ -7,7 +7,15 @@ import Theme from '../../src/Theme';
 
 // Workaround to render a component and provide it with a theme
 const customRender = (node, options) => {
```

Comment:
I don't understand here. `customerRender` call itself, it will be an infinite loop ? isn't it ?

Reply:
@AntoninMarchardDev Nop it call itself only on rerender. For information about custom render: https://github.com/kentcdodds/react-testing-library#custom-render